pkg/patch/patch.go

```diff
@@ -231,13 +231,22 @@ func patchWithContext(ctx context.Context, ch chan error, image, reportFile, pat
 		return err
 	})
 
+	var c console.Console
+	if cn, err := console.ConsoleFromFile(os.Stderr); err == nil {
+		c = cn
+	}
+	mode := progressui.AutoMode
+	if log.GetLevel() >= log.DebugLevel {
+		mode = progressui.PlainMode
+	}
+	display, err := progressui.NewDisplay(c, mode)
+	if err != nil {
+		return err
+	}
```

Comment:
```suggestion
	out := os.Stderr
	if cf, _ := console.ConsoleFromFile(os.Stderr); cf != nil {
		out = cf
	}
	mode := progressui.AutoMode
	if log.GetLevel() >= log.DebugLevel {
		mode = progressui.PlainMode
	}
	display, err := progressui.NewDisplay(out, mode)
	if err != nil {
		return err
	}
```

Reply:
@cpuguy83 i removed `ConsoleFromFile` completely, wdyt?

Reply:
I think if you do that it will never do TTY mode?

Reply:
tested and works with tty (output is glitchy for whatever reason on my terminal, but it was the same before)